### PR TITLE
Update react-geo version

### PR DESCRIPTION
**Pull request by mholthausen**
This MR

* sets `react-geo` to version [15.0.0-beta.2](https://github.com/terrestris/react-geo/releases/tag/v15.0.0-beta.2) in preparation for the release of version 15
* **Note**: react-geo contains a breaking change in version 15, which is considered here by the adjustment of `tsconfig.json`

@terrestris/devs please review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,6 +91,7 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.10.4",
     "@babel/runtime": "^7.9.2",
+    "@hanreev/types-ol": "^3.4.2",
```
Comment (hwbllmnn) [inline package.json:94]: Is this change really what we want? I'd assume the original ol types are what we want to use

**Review by hwbllmnn — APPROVED** [review]
LGTM, I just have a question

File: package.json (hunk 1 above)
Comment (simonseyock) [reply package.json:94]: The `@hanreev/types-ol` are an extension of the original ol types. It just offers additional type security.

Comment (mholthausen) [reply package.json:94]: Yes, I thought if it is used in this way at `react-geo`, we should adapt it here as well. 
If there are no objections, I will merge these changes soon.

**Pull request merged**